autosar/util/dcf.py

```diff
@@ -282,7 +282,7 @@ def save_dcf(self, dest_dir, dcf_name, xml_file_map, schema = None, force = True
             lines=['<?xml version="1.0" encoding="utf-8"?>']
             if schema is None:
                 #TODO: Below line needs to be improved
-                schema_string = "{0}{1}{2}_DEV".format(*str(self.ws.version).split('.'), self.ws.patch)
+                schema_string = "{0}{1}{patch}_DEV".format(*str(self.ws.version).split('.'), patch=self.ws.patch)

```

Comment:
All we need to to is to remove the '.' from the version string and It's more readable if this line changed into
`schema_string = "{0}{1}_DEV".format(str(self.ws.version).replace('.',''), self.ws.patch)`

This section of code is temporary anyway and needs to be properly updated to work with more recent versions of AUTOSAR

Reply:
Fixed in new commit.